components/drivers/usb/usbdevice/class/rndis.c

```diff
@@ -1348,6 +1348,19 @@ ufunction_t rt_usbd_function_rndis_create(udevice_t device)
     intf_comm = rt_usbd_interface_new(device, _interface_handler);
     intf_data = rt_usbd_interface_new(device, _interface_handler);
 
+    if( (intf_comm == RT_NULL) || (intf_data == RT_NULL) )
+    {
+        LOG_E("%s,%d: no memory!", __func__, __LINE__);
+
+        if( intf_comm != RT_NULL )
```

Comment:
RT-Thread不使用这种if括号前后空格的风格

```suggestion
        if(intf_comm != RT_NULL)
```

Reply:
已修改